### ISSUE TITLE
Search only on certain sites

### ISSUE THREAD

**Issue by dimqua** [searx/searx#1008]
Please consider to add a possibility to search only on user-defined websites, see Google's Custom Search as an example.

**Comment by dalf** [comment]
What about the ```site:only-this-website.com``` ?

**Comment by dimqua** [comment]
Yes, but it's not useful if you need to search on dozen websites, since you need to type something like `site:example.com OR site:site:example2.com` manually every time.

**Comment by dalf** [comment]
Would it be good it could be set as a preference ?

[EDIT] Searx appends automaticaly ```site:...``` to each request (the value would be from the preferences).

Each engines needs to define if they can support ```site:....```

**Comment by dimqua** [comment]
Yes, if it's possible to implement.
